examples/with-zones/blog/app/layout.tsx

```diff
@@ -0,0 +1,16 @@
+export const metadata = {
+  title: "Next.js - With Zones",
```

Comment:
If the user run the example, add another visual aid to show them where they are. 

```suggestion
  title: "Next.js - Blog Zone",
```